CONTRIBUTING.md

```diff
@@ -8,6 +8,12 @@ To build the chef-dk, we use the omnibus system. Go to the
 A standard `bundle install` in the root directory and in the `omnibus/`
 directory will get you started. You can run unit tests via:
 
+## Running `chef` Locally
+
+```
+bundle exec chef
```

Comment:
For commands that require you to be in a specific working directory (such as running `chef install` with no arguments), you can set the `BUNDLE_GEMFILE` env var which will allow you to run your copy of the code from anywhere:

```
BUNDLE_GEMFILE=~/code/chef-dk/Gemfile bundle exec chef
```